src/index.d.ts

```diff
@@ -1,7 +1,8 @@
 import { Request } from "express";
 import { RequestHandler } from "express-serve-static-core";
 import { JSONSchema4, JSONSchema6, JSONSchema7 } from "json-schema";
-import Ajv, { ErrorObject, Options as AjvOptions } from "ajv";
+import { ErrorObject, Options as AjvOptions } from "ajv";
+import AjvCore from "ajv/lib/core";
```

Comment:
I've been testing this PR with a TypeScript application and I've run into a type error, which I think is caused by this using the `AjvCore` type instead of the `Ajv` type. Official Ajv plugins use the `Ajv` type, for example [here in `ajv-formats`](https://github.com/ajv-validator/ajv-formats/blob/master/src/index.ts#L10).

With this TypeScript code:

```typescript
import addFormats from "ajv-formats";
import { Validator, ValidationError } from "express-json-validator-middleware";

const validator = new Validator({
    coerceTypes: true,
});

addFormats(validator.ajv, ["date-time"])
    .addKeyword("kind")
    .addKeyword("modifier");
```

I receive this type error when running `tsc`:

```
src/lib/middleware/validation/index.ts:9:12 - error TS2345: Argument of type 'import("/home/simonplend/dev/github/simonteaching/api-project/node_modules/express-json-validator-middleware/node_modules/ajv/lib/core").default' is not assignable to parameter of type 'import("/home/simonplend/dev/github/simonteaching/api-project/node_modules/ajv-formats/node_modules/ajv/dist/core").default'.
  Types of property 'opts' are incompatible.
    Type 'import("/home/simonplend/dev/github/simonteaching/api-project/node_modules/express-json-validator-middleware/node_modules/ajv/lib/core").InstanceOptions' is not assignable to type 'import("/home/simonplend/dev/github/simonteaching/api-project/node_modules/ajv-formats/node_modules/ajv/dist/core").InstanceOptions'.
      Type 'InstanceOptions' is not assignable to type 'CurrentOptions'.
        Types of property 'keywords' are incompatible.
          Type 'import("/home/simonplend/dev/github/simonteaching/api-project/node_modules/express-json-validator-middleware/node_modules/ajv/lib/types/index").Vocabulary | undefined' is not assignable to type 'import("/home/simonplend/dev/github/simonteaching/api-project/node_modules/ajv-formats/node_modules/ajv/dist/types/index").Vocabulary | undefined'.
            Type 'Vocabulary' is not assignable to type '(string | KeywordDefinition)[]'.
              Type 'string | import("/home/simonplend/dev/github/simonteaching/api-project/node_modules/express-json-validator-middleware/node_modules/ajv/lib/types/index").KeywordDefinition' is not assignable to type 'string | import("/home/simonplend/dev/github/simonteaching/api-project/node_modules/ajv-formats/node_modules/ajv/dist/types/index").KeywordDefinition'.
                Type 'CodeKeywordDefinition' is not assignable to type 'string | KeywordDefinition'.
                  Type 'import("/home/simonplend/dev/github/simonteaching/api-project/node_modules/express-json-validator-middleware/node_modules/ajv/lib/types/index").CodeKeywordDefinition' is not assignable to type 'import("/home/simonplend/dev/github/simonteaching/api-project/node_modules/ajv-formats/node_modules/ajv/dist/types/index").CodeKeywordDefinition'.
                    Types of property 'code' are incompatible.
                      Type '(cxt: import("/home/simonplend/dev/github/simonteaching/api-project/node_modules/express-json-validator-middleware/node_modules/ajv/lib/compile/validate/index").KeywordCxt, ruleType?: string | undefined) => void' is not assignable to type '(cxt: import("/home/simonplend/dev/github/simonteaching/api-project/node_modules/ajv-formats/node_modules/ajv/dist/compile/validate/index").KeywordCxt, ruleType?: string | undefined) => void'.
                        Types of parameters 'cxt' and 'cxt' are incompatible.
                          Type 'import("/home/simonplend/dev/github/simonteaching/api-project/node_modules/ajv-formats/node_modules/ajv/dist/compile/validate/index").KeywordCxt' is not assignable to type 'import("/home/simonplend/dev/github/simonteaching/api-project/node_modules/express-json-validator-middleware/node_modules/ajv/lib/compile/validate/index").KeywordCxt'.
                            The types of 'gen._scope' are incompatible between these types.
                              Type 'import("/home/simonplend/dev/github/simonteaching/api-project/node_modules/ajv-formats/node_modules/ajv/dist/compile/codegen/scope").Scope' is not assignable to type 'import("/home/simonplend/dev/github/simonteaching/api-project/node_modules/express-json-validator-middleware/node_modules/ajv/lib/compile/codegen/scope").Scope'.
                                Property '_names' is protected but type 'Scope' is not a class derived from 'Scope'.

9 addFormats(validator.ajv, ["date-time"])
             ~~~~~~~~~~~~~
```

Can this be changed back to importing the `Ajv` type, and then that type be used in the constructor instead of `AjvCore`?